src/libs/antares/study/fwd.h

```diff
@@ -97,21 +97,19 @@ enum StudyError
 };
 
 /*!
-** \brief Types of Study
+** \brief Types of Simulation
 */
-enum StudyMode
+enum class SimulationMode
 {
-    stdmUnknown = -1,
     //! Economic
-    stdmEconomy,
+    Economy,
     //! Adequacy (subset of the economy mode)
-    stdmAdequacy,
+    Adequacy,
     //! Expansion
-    stdmExpansion,
```

Comment:
Not necessarily for that PR, but should we rename it something more generic, like "ProblemsGeneration" ? I don't know enough the specifics of this mode to find an appropriate name though.

Reply:
Yes, this may be renamed